cfgov/v1/forms.py

```diff
@@ -1,31 +1,53 @@
 from collections import Counter
-
 from django import forms
 from django.db.models import Q
 from django.forms import widgets
 from taggit.models import Tag
 
 from .models.base import Feedback
 from v1.util.categories import clean_categories
-from v1.util import ref
-
-
-class FilterDateField(forms.DateField):
-    def clean(self, value):
-        from sheerlike.templates import get_date_obj
-        if value:
-            try:
-                value = get_date_obj(value)
-            except Exception as e:
-                pass
-        return value
+from v1.util import ERROR_MESSAGES, ref
 
 
 class MultipleChoiceFieldNoValidation(forms.MultipleChoiceField):
     def validate(self, value):
         pass
 
 
+class FilterableDateField(forms.DateField):
+    default_input_formats = (
+        '%m/%d/%Y',
+    )
+
+    default_widget_attrs = {
+        'class': 'js-filter_range-date',
+        'type': 'text',
+        'placeholder': 'mm/dd/yyyy',
+        'data-type': 'date'
+    }
+
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault('required', False)
+        kwargs.setdefault('input_formats', self.default_input_formats)
+        kwargs.setdefault('widget', widgets.DateInput(
+            attrs=self.default_widget_attrs
+        ))
+        kwargs.setdefault('error_messages', ERROR_MESSAGES['DATE_ERRORS'])
+        super(FilterableDateField, self).__init__(*args, **kwargs)
+
+
+class FilterableFromDateField(FilterableDateField):
+    def __init__(self, *args, **kwargs):
```

Comment:
@rosskarchner that's true, although if any other form used this field we'd have to provide the same argument. I was thinking it made sense for this CSS to live as part of the field class, since it's field-specific not really form-specific.